deviceinfo.js

```diff
@@ -176,7 +176,7 @@ const devicesWithNotch = [
   },
   {
     brand: 'xiaomi',
-    model: 'POCO F1',
+    model: 'POCOPHONE F1',
```

Comment:
Hi, could you add another line instead of changing this one?

Some phones will report different model names depending on the country/operator/etc.

Reply:
Hi @machour, thanks for the suggestion. Pull request updated.